.github/workflows/update-geolite-database.yml

```diff
@@ -0,0 +1,106 @@
+name: Update GeoLite Database
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Target branch against which to create PR'
+        required: false
+        default: 'master'
+
+env:
+  MAXMIND_URL: 'https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country&license_key=${{ env.MAXMIND_LICENSE_KEY }}&suffix=tar.gz'
+  MAXMIND_SHA256_URL: 'https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country&license_key=${{ env.MAXMIND_LICENSE_KEY }}&suffix=tar.gz.sha256'
+  TAR_FILE_NAME: 'GeoLite2-Country.tar.gz'
+  TAR_SHA256_FILE_NAME: 'GeoLite2-Country.tar.gz.sha256'
+  TAR_UNZIPPED_ROOT_PATTERN: 'GeoLite2-Country_*'
+  DB_FILE: 'GeoLite2-Country.mmdb'
+  DB_DESTINATION_PATH: 'common/static/data/geoip'
+
+jobs:
+  download-and-replace:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Download GeoLite tar file
+        run: |
+          wget -O '${{ env.TAR_FILE_NAME }}' '${{ env.MAXMIND_URL }}'\
+
+      - name: Download GeoLite sha256 file
+        run: |
+          wget -O '${{ env.TAR_SHA256_FILE_NAME }}' '${{ env.MAXMIND_SHA256_URL }}'\
+
+      - name: Check SHA256 hash
+        run: |
+          sha256sum '${{ env.TAR_FILE_NAME }}' | grep $(cat '${{ env.TAR_SHA256_FILE_NAME }}' | cut -d' ' -f1)
+
+      - name: Extract tar file
+        run: |
+          tar xvf '${{ env.TAR_FILE_NAME }}'
+
+      - name: Copy DB file to destination path
+        run: |
+          find . -type d -name '${{ env.TAR_UNZIPPED_ROOT_PATTERN }}' -exec cp {}/'${{ env.DB_FILE }}' '${{ env.DB_DESTINATION_PATH }}'/ \;
+
+      - name: Delete un-required content
+        run: |
+          rm '${{ env.TAR_FILE_NAME }}'
+          rm '${{ env.TAR_SHA256_FILE_NAME }}'
+          find . -type d -name '${{ env.TAR_UNZIPPED_ROOT_PATTERN }}' -exec rm -r {} \; || true
+
+      - name: PR preflight
+        run: |
+          if git diff --exit-code; then
+              echo 'Summary: No updates/changes detected. Terminating the run and no pull request is going to be created.' | tee -a '$GITHUB_STEP_SUMMARY'
+              exit 1
+          else
+              echo 'Updates/changes detected, going to create PR.'
+          fi
+
+      - name: Make a PR
+        id: make-pr
+        uses: peter-evans/create-pull-request@v6
```

Comment:
You don't need to use a 3rd party tool, you can just use the GitHub CLI which is available by default.

See https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows for some examples.

You should be able to just call the `gh pr create...` with all the relevant information.